statesync/stateprovider.go

```diff
@@ -174,12 +174,6 @@ func (s *lightClientStateProvider) State(ctx context.Context, height uint64) (sm
 		return sm.State{}, err
 	}
 
-	// VRF proof
-	proofHash, err := vrf.ProofToHash(lastLightBlock.Proof.Bytes())
-	if err != nil {
-		return sm.State{}, err
-	}
-
 	state.Version = tmstate.Version{
```

Comment:
I think there is not a coverage about `State` function. 
How about adding it if possible? ^^a

Reply:
It seem to be hard, but I'll try it.

Reply:
No, it need many time. I'll fix it in another PR.